api/app/Observers/PoolCandidateObserver.php

```diff
@@ -46,7 +46,7 @@ public function updated(PoolCandidate $poolCandidate): void
                     ));
                 } catch (Throwable $e) {
                     // best-effort: log and continue
-                    Log::error('Failed to send "application status changed" notification to ['.$poolCandidate->id.']');
+                    Log::error('Failed to send "application status changed" notification to ['.$poolCandidate->id.']'.$e->getMessage());
```

Comment:
Nit pick. but maybe a space here? :sweat_smile: 
```suggestion
                    Log::error('Failed to send "application status changed" notification to ['.$poolCandidate->id.'] '.$e->getMessage());
```